easybuild/toolchains/mpi/craympich.py

```diff
@@ -54,6 +55,7 @@ class CrayMPICH(Mpi):
         '_opt_MPICXX': '',
         '_opt_MPIF77': '',
         '_opt_MPIF90': '',
+        '_opt_MPIFC': '',
```

Comment:
See `easybuild/toolchains/mpi/mpich.py` below: There you use a dict and `MPI_COMPILER_VARIABLES`. Consistency would be nice and using a dict is probably less error prone.